### DISCUSSION TITLE
Can't see any event on Facebook event manager

### DISCUSSION THREAD

**Discussion by lkorczowski** [oddbit/flutter_facebook_app_events#134] [Q&A]
Hi !

With @mazenhalawi, we are trying to log events but unfortunately we couldn't succeed yet.

The Facebook SDK seems to work : 
![Screenshot 2021-09-03 at 12 23 55](https://user-images.githubusercontent.com/15196605/131991189-1c00e84f-5c5b-49b4-9db1-05db32ace200.png)
it seems that we are receiving standard evens that come with the SDK (3 events : Activate App, App Installs, Complete Registration). Note that we didn't set up anything to receive these events, and (because of that?) we have the "Missing fb_mobile_complete_registration Currency and Value Parameters" error for complete registration. But that doesn't seems the issue here, isn't it ?

Then we tried to use both Facebook event manager and test to see if we can log event without success with very simple tests, e.g. when someone click on Forgotten password
![Screenshot 2021-09-03 at 12 39 33](https://user-images.githubusercontent.com/15196605/131995118-24fac2ca-be12-41e2-96b5-bd308246f5b2.png)

As you can see here, we also log to firebase and firestore in order to double check (and we are receiving it properly). We test with or without additional parameters without success.

Could you help us? Do you have ideas ?

Flutter: 2.0.3 (null-safety deployed)
```
[✓] Flutter (Channel stable, 2.0.3, on macOS 11.5.2 20G95 darwin-x64, locale en-FR)
    • Flutter version 2.0.3 at /Users/louis/flutter
    • Framework revision 4d7946a68d (6 months ago), 2021-03-18 17:24:33 -0700
    • Engine revision 3459eb2436
    • Dart version 2.12.2

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/louis/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Library/Java/JavaVirtualMachines/jdk-14.0.1.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment (build 14.0.1+7)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] IntelliJ IDEA Community Edition (version 2020.3.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 54.1.4
    • Dart plugin version 203.7759

[✓] Connected device (1 available)
    • M2002J9G (mobile) • 9550ae8 • android-arm64 • Android 11 (API 30)

! Doctor found issues in 4 categories.
```

**Comment by tjerkw** [comment]
Same problem, zero events shown in Facebook Events manager. Also not in the "Test events" tab. Both for Android and iOS

**Comment by DennisAlund** [comment]
Could this be related and helped by the discussions in #122 ?

**Comment by tjerkw** [comment]
I just released my app hoping it would work. And it did. Weirdly enough.

On Wed, 15 Sep 2021 at 03:26, Dennis Alund ***@***.***> wrote:

> Could this be related and helped by the discussions in #122
> <https://github.com/oddbit/flutter_facebook_app_events/discussions/122> ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/oddbit/flutter_facebook_app_events/discussions/134#discussioncomment-1330579>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGH5MKWCZWV4WZUD2NLOF3UB7Y25ANCNFSM5DLQW3CA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
--
Tjerk Wolterink

**Comment by santiLabhouse** [comment]
Hi, I did all the steps: added all in info.plist (FacebookAppID, FacebookClientToken, FacebookAdvertiserIDCollectionEnabled, CFBundleURLSchemes, etc), I request the app transparency tracking inside the app, and also upgraded FacebookSDK to latest version, but still I don't receive events in the test event console. It's strange because I can see some standard events logged (like app install or activated app) in the general info tab on facebook business manager, but others like Subscription or Purchase are not showing. And I can't test in real time because I can't see any event logging in the test event console. Someone knows why this could be happening? Thanks

**Reply by samatzp** [reply]
Hey! Did you find the solution?
